Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+# Configuring OSVR HDK2 with Vive Puck Tracking
+
+This document outlines how to add room-scale positional tracking to HDK2 by mounting an HTC Vive tracking puck and using the Lighthouse base stations and tracking system.
+
+Hardware Requirements:
+OSVR HDK2 with mount
+Vive Tracking Puck
+Lighthouse base stations
+USB - microUSB cable
+
+
+Software Requirements:
+OSVR-Core build
```

Comment:
Bulleted list? Also omit `build`.

Is SteamVR really optional? Or did you mean that SteamVR-OSVR is optional?